src/event/handlers/CommandParserHandler.ts

```diff
@@ -25,6 +25,8 @@ class CommandParserHandler extends EventHandler {
 			const args = message.content.replace("?", "").split(" ");
 			const trigger = args.shift() || args[0];
 
+			if (!trigger) return;
```

Comment:
This check could have been done in many other ways, such as checking if message's length is bigger than 1 before continuing, so it would stop if message was `"?"`. However, I think this more consistent.